community-operators/cluster-logging/clusterlogging.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,417 @@
+#! validate-crd: deploy/chart/templates/0000_30_02-clusterserviceversion.crd.yaml
+#! parse-kind: ClusterServiceVersion
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: clusterlogging.v0.0.1
+  namespace: placeholder
+  annotations:
+    categories: "openshift, optional"
```

Comment:
I believe `openshift optional` is a single, distinct category, so there's no need for a comma there. I would also add a category like "logging" or "metrics".